### PR TITLE
nordic: pkg installs plasma styles to the wrong directory

### PR DESCRIPTION
**Pull request by jkoking**
## Description of changes

Rearranged the nordic package directory structure

update 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by romildo** [comment]
Are you sure of that? Kde is not my main DE. But ...
- why the author provides the sub directory with the name `look-and-feel`?
- looking at the `share/plasma/look-and-feel` directories provided by other packages, one can find similar file contents:
  ```console
  $ find -L /run/current-system/sw/share/plasma/look-and-feel/ -name contents -or -name metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia-light/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia-light/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/org.kde.breezedark.desktop/contents
  /run/current-system/sw/share/plasma/look-and-feel/Nordic/contents
  /run/current-system/sw/share/plasma/look-and-feel/Nordic/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/org.kde.breeze.desktop/contents
  /run/current-system/sw/share/plasma/look-and-feel/Nordic-darker/contents
  /run/current-system/sw/share/plasma/look-and-feel/Nordic-darker/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc-dark/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc-dark/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/org.kde.oxygen/contents
  /run/current-system/sw/share/plasma/look-and-feel/org.kde.oxygen/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord/contents
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia-dark/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia-dark/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/org.kde.breezetwilight.desktop/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc-darker/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc-darker/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Nordic-bluish/contents
  /run/current-system/sw/share/plasma/look-and-feel/Nordic-bluish/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur-dark/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur-dark/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.arc/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-light/contents
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-light/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur-alt/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.vinceliuice.WhiteSur-alt/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia/contents
  /run/current-system/sw/share/plasma/look-and-feel/com.github.varlesh.materia/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-solid/contents
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-solid/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-light-solid/contents
  /run/current-system/sw/share/plasma/look-and-feel/Utterly-Nord-light-solid/metadata.desktop
  /run/current-system/sw/share/plasma/look-and-feel/Dracula/contents
  /run/current-system/sw/share/plasma/look-and-feel/Dracula/metadata.desktop
  ````

**Comment by jkoking** [comment]
The documentation says plasma styles go in `plasma/look and feel/desktoptheme`

- documentation in questions: https://develop.kde.org/docs/plasma/theme/theme-details/

 - I made this pull request because the Nordic plasma style did not appear in system settings /appearance/ plasmastyle

 -  My previous commit added the Nordic themes to plasma styles but it also remove them from the global themes and splash screen tabs I'm trying to figure out how to fix this

**Comment by jkoking** [comment]
Fixed, global theme and splash screen now appear i had to move them back to look-and-feel and put a copy of metadata.desktop there. Everything else stays in desktoptheme.

**Comment by romildo** [comment]
Your commit messages is not following the nixpkgs guidelines. Change to something like the following like:
```
nordic: fix installation directories of plasma styles
```

File: pkgs/data/themes/nordic/default.nix [hunk 1]
```diff
@@ -121,10 +121,15 @@ stdenvNoCC.mkDerivation rec {
...
     mv -v $out/share/themes/Nordic/kde/kvantum/* $out/share/Kvantum/
-    mv -v $out/share/themes/Nordic/kde/plasma/look-and-feel $out/share/plasma/
+    cp -vr $out/share/themes/Nordic/kde/plasma/look-and-feel $out/share/plasma/look-and-feel/
+    mv -v $out/share/themes/Nordic/kde/plasma/look-and-feel $out/share/plasma/desktoptheme/
     mv -v $out/share/themes/Nordic/kde/folders/* $out/share/icons/
     mv -v $out/share/themes/Nordic/kde/cursors/*-cursors $out/share/icons/
 
+
+  rm -rf $out/share/plasma/look-and-feel/*/contents/{logout,osd,components}
+  rm -rf $out/share/plasma/desktoptheme/*/contents/{{defaults,splash,previews}
```
Comment (romildo) [inline pkgs/data/themes/nordic/default.nix:131]:
```suggestion
    rm -rf $out/share/plasma/look-and-feel/*/contents/{logout,osd,components}
    rm -rf $out/share/plasma/desktoptheme/*/contents/{{defaults,splash,previews}
```

**Review by romildo — CHANGES_REQUESTED** [review]

**Review by romildo — APPROVED** [review]
LGTM

**Pull request merged**